src/Sylius/Bundle/CoreBundle/Provider/DefaultTaxZoneProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Provider;
+
+use Sylius\Bundle\SettingsBundle\Model\Settings;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class DefaultTaxZoneProvider implements DefaultTaxZoneProviderInterface
+{
+    /**
+     * @var Settings
+     */
+    private $settings;
+
+    /**
+     * @param Settings $settings
+     */
+    public function __construct(Settings $settings)
+    {
+        $this->settings = $settings;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function provide()
```

Comment:
I am not sure about this Provider -> provide naming... Not saying that ``DefaultTaxZoneProvider::getZone/getTaxZone/getDefaultTaxZone`` is better but I don't like both actually. :D